usr/src/uts/common/brand/lx/os/lx_misc.c

```diff
@@ -613,8 +623,9 @@ lx_forklwp(klwp_t *srclwp, klwp_t *dstlwp)
  */
 /* ARGSUSED */
 static void
-lx_save(klwp_t *t)
+lx_save(void *arg)
 {
+	klwp_t *t = (klwp_t *)arg;
```

Comment:
Unused variable - causes DEBUG build failure.
In OmniOS I did this to keep it around for future reference.
```suggestion
	klwp_t *t __unused = (klwp_t *)arg;
```

Reply:
Well shoot.  I should just trash this whole line then.  Lemme build to confirm.